posts-report.php

```diff
@@ -8,6 +8,9 @@
 
 use Timber\Timber;
 
+$context = Timber::get_context();
+$context['new_ia'] = !empty(planet4_get_option('new_ia'));
```

Comment:
Doesn't seem to work for me, as the Timber::render on l.14 is not called with $context, and new_ia is null in the template. This works on my local:
```php
$context['spinner_gif'] = get_template_directory_uri() . '/images/wpspin_light-2x.gif';
Timber::render([ 'posts_report.twig' ], $context);
```

Reply:
That's weird, it used to work for me on local but indeed not anymore 😅 I see that it doesn't work on the test instance either... Code updated 👍